From: =?utf-8?b?0JLQu9Cw0YHQtdC90LrQviDQnNC40YXQsNC40Lsg0JLQuNC60YLQvtGA?=
 =?utf-8?b?0L7QstC40Yc=?= <Michael1972@mail.ru>
Date: Tue, 23 Jan 2018 00:00:00 +0100
Subject: COMBINING GREEK YPOGEGRAMMENI is not alphanumeric

The U+0345 COMBINING GREEK YPOGEGRAMMENI Unicode character is a diacritical
mark, and not an alphanumeric character.

Bugs-Debian: https://bugs.debian.org/785640
Bugs-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/dictd/+bug/1744896
---
 iswalnum.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/iswalnum.c b/iswalnum.c
index 10599cb..1e72d4a 100644
--- a/iswalnum.c
+++ b/iswalnum.c
@@ -4,7 +4,7 @@ static const wint_t start [] = {
      0,
     48,    65,    97,   170,   181,   186,   192,   216,
    248,   546,   592,   688,   699,   720,   736,   750,
-   837,   890,   902,   904,   908,   910,   931,   976,
+   890,   902,   904,   908,   910,   931,   976,
    986,  1024,  1164,  1223,  1227,  1232,  1272,  1329,
   1369,  1377,  1488,  1520,  1569,  1600,  1632,  1649,
   1749,  1765,  1776,  1808,  1810,  1920,  2309,  2365,
@@ -50,7 +50,7 @@ static int count [] = {
      0,
     10,    26,    26,     1,     1,     1,    23,    31,
    296,    18,    94,     9,     7,     2,     5,     1,
-     1,     1,     1,     3,     1,    20,    44,     8,
+     1,     1,     3,     1,    20,    44,     8,
     28,   130,    57,     2,     2,    38,     2,    38,
      1,    39,    27,     3,    26,    11,    10,    99,
      1,     2,    13,     1,    27,    38,    53,     1,
